### ISSUE TITLE
Phase-Out Browser Baseplate

### ISSUE THREAD

**Issue by Foggalong** [numixproject/numix-core#5153]
The browser baseplate we use across a wide range of icons is starting to look significantly more dated than it did however many years ago I chucked it together. This issue is a general note to me (or to anyone else who fancies a go) to try and come up with an alternative which ideally:

1. Can still be used in a wide variety of icons
2. Looks nicer while not just focusing in Europe or America
3. Isn't a full baseplate design, since those are being phased out generally

**Comment by palob** [comment]
I was about to open an issue about this for technical reasons.

**Comment by Foggalong** [comment]
Are the technical reasons how ridiculously complicated it is? :joy:

**Comment by maxtron95** [comment]
Well there is a lot of web browser icons (including firefox) that is using the browser baseplate. I think the firefox icon looks weird as a full baseplate square icon in my opinion.

**Comment by Foggalong** [comment]
@creepertron95 Yeah full baseplate icons are being phased out generally, the browser one in particular though is one which has been niggling me because it's a poor (and complex) design.

**Comment by maxtron95** [comment]
These are the icons that are currently using the browser baseplate:

- aarddict
- abrowser
- drawpile
- eiskaltdcpp
- empire-total-war
- ephemeral
- eric-web
- firefox-icon-unbranded
- gottengeography
- jamulus
- luakit
- midori
- mlclient
- mlterm
- nmap
- ooo-web
- pac
- parley
- pglgui
- qmaptool
- qomp
- qrestclient
- qutebrowser
- smtube
- smuxi-frontend-gnome
- web-browser
- zeroinstall

**Comment by maxtron95** [comment]
@palob Seems we can ditch the globe on the `komikku` icon. The icon has just been redesigned since you last made a PR for it in #5053.

![info febvre Komikku](https://user-images.githubusercontent.com/53346722/81417357-ab707680-9110-11ea-845b-0ff7a2a7ae86.png)

**Comment by palob** [comment]
The lowest hanging fruit-solution would be a (symbol-on-baseplate) globe with fantasy continents on it. If we ditch the fullscreen design, we need to find ways to incorporate the on-globe-baseplate-symbols.

Right now we use the globe design for different kinds of applications: web browsers, map/navigation apps, communication apps, localisation apps, games.

There are other widely-used ambiguous metaphors, for instance a compass but maybe we won't be able to find a new one which fits all.

Question is if the globe metaphor is a good one for all the apps in the first place.

**Comment by Foggalong** [comment]
A globe with fantasy continents sounds good, but yeah it's going to have to be a more careful process of seeing how that integrates into each of the existing icons rather than a fix-all situation.

**Comment by palob** [comment]
GNOME Web/Epiphany recently got a new icon. Maybe something to draw inspiration from.
https://gitlab.gnome.org/GNOME/epiphany/-/commit/a308ad5a61b63443214b65a388d9910573567d1b

**Comment by Foggalong** [comment]
I like that idea, though it uses quite a few colours to make it work - could prove difficult to integrate into some of our browser designs

**Comment by palob** [comment]
It could work with just altering saturation and lightness while keeping the hue constant.
Strictly speaking the shading would have to be different for Circle and Square, not sure though if that's neccessary.
In cases where there's two different colours for the oceans and the continents we could work with those 2 colours.

If we depart from a full-baseplate design (which we'll have to in case of the Epiphany approach because it doesn't work on the Square baseplate) we'll maybe run into problems with the icons composed of world baseplate + symbol on it.
Such symbols might have to be shrinked but I think it's worth trying.

**Comment by palob** [comment]
In addition to the above list we've got some more icons with a globe symbol on a (non-globe) baseplate.

**Comment by palob** [comment]
``browser-tor`` is done.
